hive/src/test/java/org/apache/iceberg/hive/HiveTableBaseTest.java

```diff
@@ -66,6 +66,7 @@ public void dropTestTable() throws Exception {
     tableLocation.getFileSystem(hiveConf).delete(tableLocation, true);
     catalog.dropTable(TABLE_IDENTIFIER, false /* metadata only, location was already deleted */);
   }
+
```

Comment:
Nit: this file doesn't need to change. Can you revert this to avoid git conflicts?

Reply:
Done